fdbt-dev/Makefile

```diff
@@ -155,6 +155,9 @@ generate-validate-p2p-period: generate-p2p-period validate-latest-file
 generate-flat-fare-geo-zone:
 	./scripts/trigger_netex_convertor.sh flatFareGeoZone
 
+generate-validate-geo-zone-with-exceptions:
```

Comment:
This wont validate anything, this will only generate. Check each fare type, they each have 2 commands.